flixel/input/gamepad/ButtonID.hx

```diff
@@ -0,0 +1,57 @@
+package flixel.input.gamepad;
+
+import flixel.system.macros.FlxMacroUtil;
+
+/**
+ * Maps enum values and strings to integer keycodes.
+ */
+@:enum
+abstract ButtonID(Int) from Int to Int
```

Comment:
Not sure I like this name... guess `FlxGamepadButton` already exists... `FlxGamepadButtonID` is rather long.